docs/product-assembly_dm-on-openshift/src/main/asciidoc/dm-openshift-prepare-con.adoc

```diff
@@ -1,10 +1,15 @@
 [id='dm-openshift-prepare-con']
```

Comment:
I did consider this. However, the OpenShift processes for DM and PAM are entirely different, which is why I took this decision. It *may* be that in DM 7.1 they will bring the process closer to PAM, in which case I will use this approach.

However, the assembly will in fact not be built. While I will keep it *for now* as a directory (simply because moving it to a guide directory will make it impossible to develop under the current ime constraints), I will create *three* separate assemblies for the authoring, managed server, and immutable server processes. I will not put "pam" in the directory names, in case DM moves to PAM direction (which seems somewhat likely). Mut I will not make any attempt to make it applicable to DM simply because the processes, as they are, are way, way to different.